docs/src/pages/docs/functions/helpers.md

```diff
@@ -283,6 +283,66 @@ that can be called in any combination, such as: `f(x, y, z)`, `f(x)(y)(z)`,
 `f(x, y)(z)`, or even `f(x)(y, z)`. This is great for doing partial application
 on functions for maximum re-usability.
 
+```javascript
+import curry from 'crocks/helpers/curry'
+
+// add :: (Number, Number, Number) -> Number
+const add = (a, b, c) =>
+  a + b + c
+
+// partial application impossible
+add(1)
+// => 1 + undefined + undefined => NaN
+
+// NOT the only possible type signature as highlighted below
+// curriedAdd :: Number -> Number -> Number -> Number
+const curriedAdd = curry( add )
+
+curriedAdd(1)
+// => [Function: curried]
+
+// Number -> Number -> Number -> Number
+curriedAdd(1)(2)(3)
+// => 6
+
+// (Number, Number) -> Number -> Number
```

Comment:
Should be a comment under the result, maybe using simple variable names instead of types like:
// (a, b)(c)

Reply:
> ```js
> // pluck :: String -> [ a ] -> Maybe b
> const pluck = curry(
>   compose(map, prop)
> )
> 
> const data = [
>   { a: 'nice' },
>   { a: 'great', b: 'nice' },
>   { b: 'nice' },
> ]
> 
> pluck('a', data).map(x => `${x}`)
> //=> [ Just 'nice', Just 'great', Nothing ]
> ```

Should I include the signatures for `map`, `prop`, and `compose` and show the contrast of using `compose(map, prop)` directly without the aid of `curry` first?

Reply:
Nah, should be fine as it is. We can iterate on it, if need be.
We should remove the `.map` bit though, so sorry I thought I removed that.

Reply:
Without the map, the proper output is gigantic. The output comment below the call to pluck would be figurative, not literal. Are you sure?

Reply:
The console output is not what is displayed in those results (`\\=>`), but a representation of the result.
We would have to put that on every result, so it is best to leave that off and just go with:

```js
pluck('a', data)
//=>  [ Just "nice", Just "great", Nothing ]
```

Reply:
OK